assembl/static2/css/views/administration.scss

```diff
@@ -435,3 +436,7 @@
     padding-bottom: 10px;
   }
 }
+
+.helper {
```

Comment:
.margin-left-20 would be a better name even though we should try to avoid to accumulate these classes I guess